my-app/functions-python/main.py

```
@@ -91,4 +95,198 @@ def deleteUserAccount(req: https_fn.CallableRequest):
+    return summary
+
+
+@scheduler_fn.on_schedule(
+    # Cron: minute hour day-of-month month day-of-week
+    # This runs at 07:00 every day in America/New_York.
+    schedule="every day 10:00",
+    region="us-central1",
+    memory=512,
+    timeout_sec=300,
+)
+def updateDeliveriesDaily(event: scheduler_fn.ScheduledEvent) -> None:
```

Ruff (0.12.2)

Unused function argument: `event`

(ARG001)